forte/mrdsrg-helper/dsrg_source.h

```diff
@@ -50,6 +50,29 @@ class DSRG_SOURCE {
     /// Renormalize denominator
     virtual double compute_renormalized_denominator(const double& D) = 0;
 
+    double compute_denominator(const double& D, int i) {
+        if (std::fabs(D) < 1e-6) {
+            switch(i) {
+                case 1:
+                    return 0;
+                case 2:
+                    return s_;
+            }
+        }
+        else {
+            switch(i) {
+                case 1:
+                    return (1.0 - std::exp(-s_ * D * D)) / D;
+                case 2:
+                    return (1.0 - std::exp(-s_ * D * D)) / D / D;
+            }
+        }
+    }
+
+    double get_s() {
```

Comment:
I don't think this is useful because s is set from outside. But if you insist to keep it, please change to
```suggestion
    double s() {
```

Reply:
I will check it.